pages/common/gh-pr.md

```diff
@@ -10,7 +10,7 @@
 
 - Check out a specific pull request locally:
 
-`gh pr checkout {{pr_number}}`
+`gh pr checkout {{1..infinity}}`
```

Comment:
this is a shorthand for `git checkout <branch_name>`. I'm sure that only 1 PR number is allowed here because Git can't checkout more than 1 branch at the same time.
```suggestion
`gh pr checkout {{pr_number}}`
```

Reply:
Range syntax is for exactly one number, not several ones. It's not an ellipsis.